lib/kennel/syncer.rb

```diff
@@ -147,18 +144,19 @@ def validate_expected_id_not_missing(expected)
 
     def print_changes(step, list, color)
       return if list.empty?
-      list.each do |_, e, a, diff|
-        klass = (e ? e.class : a.fetch(:klass))
-        Kennel.out.puts Console.color(color, "#{step} #{klass.api_resource} #{e&.tracking_id || a.fetch(:tracking_id)}")
-        diff&.each { |args| Kennel.out.puts @attribute_differ.format(*args) } # only for update
+      list.each do |item|
+        Kennel.out.puts Console.color(color, "#{step} #{item.api_resource} #{item.tracking_id}")
+        if item.respond_to?(:diff)
```

Comment:
can we have nil diff instead and do `item.diff&.each` ?

Reply:
Changed to check type `== :update` for now, which is at least much better than using `respond_to?`